hazelcast/src/main/java/com/hazelcast/internal/diagnostics/DiagnosticsLogFile.java

```diff
@@ -98,7 +98,9 @@ public void write(DiagnosticsPlugin plugin) {
 
     private File newFile(int index) {
         createDirectoryIfDoesNotExist();
-        return new File(diagnostics.directory, format(fileName, index));
+        File fresh = new File(diagnostics.directory, format(fileName, index));
+        logger.info("Current diagnostics log file is " + fresh.getAbsolutePath());
```

Comment:
I would print the directory.

Diagnostics log directory is [....]

Reply:
```
INFO: [192.168.1.5]:5701 [dev] [4.1-SNAPSHOT] Diagnostics log directory is [/Users/buraksezer/Projects/hazelcast]
```

Is that OK? I'm not sure about the square brackets.

Reply:
That is good. With customer provides strings it is best to add something like [...] around it so it is easier for them to understand what they have configured.